tests/test_util.py

```diff
@@ -10,20 +10,25 @@
 from pex.compatibility import nested, to_bytes
 from pex.pex import PEX
 from pex.pex_builder import PEXBuilder
+from pex.typing import TYPE_CHECKING
 from pex.util import CacheHelper, DistributionHelper, iter_pth_paths, named_temporary_file
 
 try:
     from unittest import mock
 except ImportError:
     import mock  # type: ignore[no-redef]
 
+if TYPE_CHECKING:
+    from typing import Any
+
 
 def test_hash():
-    empty_hash = sha1().hexdigest()
+    # type: () -> None
+    empty_hash_digest = sha1().hexdigest()
```

Comment:
MyPy didn't like that the variable was being reassigned later with a different type.